src/timessquare/dependencies/requestcontext.py

```diff
@@ -71,7 +81,8 @@ async def context_dependency(
     request: Request,
     response: Response,
     logger: BoundLogger = Depends(logger_dependency),
```

Comment:
Consider also changing this to `auth_logger_dependency` if this will always be behind Gafaelfawr.